src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingControllers/EventsControllerTests.cs

```diff
@@ -349,22 +386,22 @@ public async void Get_ValidRequest_ReturnsListOfEventsAndNextUrl()
 
             /// <summary>
             /// Scenario:
-            ///   Get events with  a an after parameter.
+            ///   TTD org Get events with  a valid set of query parameters
             /// Expected result:
-            ///   Next header contains new guid in after parameter
+            ///   Returns a list of events and a next header
             /// Success criteria:
-            ///   Next header is corrcect.
+            ///   The response has correct count. Next header is corrcect.
             /// </summary>
             [Fact]
-            public async void Get_AfterIncludedInQuery_ReturnsNextHeaderWithReplacesAfterParameter()
+            public async void Get_ValidRequest_ForTTD_ReturnsListOfEventsAndNextUrl()
```

Comment:
do we loose some infromation about what we are actually testing here? Should it be included in the scenario that the main point is to verify that the after parameter is successfully replaced?

Reply:
agree. Updated